presto-product-tests/conf/tempto/tempto-configuration-for-docker-default.yaml

```diff
@@ -4,3 +4,8 @@ databases:
   presto:
     host: presto-master
     configured_hdfs_user: hive
+  cassandra:
+    host: cassandra
+    port: 9042
+    default_schema: tpch
```

Comment:
It seems likely to me that many tests will use tables that are outside the TPCH schema.  It might be better to call this `default` or `test`.